phalcon/assets/manager.zep

```diff
@@ -488,6 +488,16 @@ class Manager
 					let prefixedPath = path;
 				}
 
+                let version = collection->getVersion();
+				if $resource->getVersion() == null && version != null {
+					if version {
+						if version === true && local {
+							let version = filemtime($resource->getRealSourcePath());
+						}
+						let prefixedPath = prefixedPath . "?ver=" . version;
+					}
+				}
```

Comment:
i find this PR very useful. maybe if this pasta is ugly, we can like in volt services..

```

$di->set('assets', function(){
  $cls = new Assets(); 
  $cls->setAppendString("?ver=".$config->assetsVersion);
  
  return $cls;
}

```

but as i said, versioning to assets its very important for me.